### PR TITLE
basic styles and tag cleaning

### PR DESCRIPTION
**Pull request by Marvizm**

### CODE REVIEW

File: components/AccountWidget/AccountWidget.jsx [hunk 1]
```diff
@@ -7,18 +7,18 @@ import { Link } from 'react-router';
 
 // Maybe we can factor out this ternary into a function?
 const AccountWidget = ({ loading, error, player }) => (
-  <div>
+  <menu>
```
Comment (howardchung) [inline components/AccountWidget/AccountWidget.jsx]: is this an actual HTML tag?

File: index.html [hunk 2]
```diff
@@ -24,7 +24,9 @@
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="msapplication-TileImage" content="/public/images/mstile-144x144.png">
     <meta name="msapplication-config" content="/public/images/browserconfig.xml">
-    <meta name="theme-color" content="#ffffff"> </head>
+    <meta name="theme-color" content="#ffffff"> 
+    <link rel="stylesheet" href="yasp.css">
```
Comment (howardchung) [inline index.html]: should be bundled via webpack

**Comment by howardchung** [comment]
@mike-robertson should review and ensure it merges cleanly with his change

File: components/AccountWidget/AccountWidget.jsx (hunk 1 above)
Comment (gu3st) [inline components/AccountWidget/AccountWidget.jsx]: Yeah. It's part of HTML5.  There's a bunch of them. section, article, header, footer (and more).  They're all just block-level elements (divs) but hold semantic value for search engines (not that the SPA will work with search engines, unless we attempt to implement server-side rendering at a later date).

**Comment by howardchung** [comment]
Rebase against master if you want to merge this

**Pull request closed without merging**